parsimonious/expressions.py

```diff
@@ -151,10 +161,32 @@ def _uncached_match(self, text, pos, cache, error):
         if text.startswith(self.literal, pos):
             return Node(self.name, text, pos, pos + len(self.literal))
 
+    def _uncached_match_list(self, token_list, pos, cache, error):
```

Comment:
That is silly, I don't remember why I wrote it like that.

As you can see, what it does is to assume that a "Token" instance also has a "text" attribute (containing, for instance, the text which the lexer put together to form the token, but could be anything, really), and compares it with that. But it has been a while, I don't remember why I needed it, or thought I needed it.

I am actually not using parsimonious for my project anymore, in the end my use case was so unique that I had to come up with a custom parsing method --- I had all sorts of requirements such as the language to be parsed changing dynamically by instructions of the language itself, and this didn't easily conform to grammar-based parsing.